application/behat.yml

```diff
@@ -5,11 +5,11 @@ default:
                 - "%paths.base%/features/authentication.feature"
                 - "%paths.base%/features/password.feature"
                 - "%paths.base%/features/user.feature"
-            contexts: [ FeatureContext ]
+            contexts: [ FeatureContext, Sil\SilIdBroker\Behat\Context\UnitTestsContext ]
```

Comment:
Interesting, I don't remember seeing multiple Contexts listed before. in the past, I've simply had the more specific context (e.g. UnitTestsContext) be a subclass of the broader contexts (FeatureContext). Do you happen to know of any advantages of listed multiple contexts vs. subclassing?

Reply:
Also, I recall a situation where it didn't make sense (or was impossible) to combine contexts into a class hierarchy. Though I can't remember the specifics, it had something to do with PHP not having multiple inheritance. If I remember or run across it again, I'll let you know.

Reply:
I think it's fine to have multiple Contexts, it was just something I hadn't done before. It sounds like in this case it was the better choice.